### PR TITLE
Configurable threshold in temperature block

### PR DESCRIPTION
**Pull request by leowzukw**
It’s pretty handy to adapt the threshold from one state to another to
your own hardware.

This is also a feature mentioned as missing in `blocks.md`.

### CODE REVIEW

File: src/blocks/temperature.rs [hunk 1]
```diff
@@ -128,13 +169,15 @@ impl Block for Temperature {
...
             }
 
-            self.text.set_state(match max {
-                0...20 => State::Good,
-                21...45 => State::Idle,
-                46...60 => State::Info,
-                61...80 => State::Warning,
-                _ => State::Critical,
-            });
+            let state =
```
Comment (atheriel) [inline src/blocks/temperature.rs]: I'd suggest moving back to the `match` syntax here. It's a little more verbose in this case but easier to reason about.

**Comment by atheriel** [comment]
Thanks, this is a nice addition. Can you take a look at the merge conflicts?

File: src/blocks/temperature.rs (hunk 1 above)
Comment (leowzukw) [reply src/blocks/temperature.rs]: Done in dd0cd52

**Comment by leowzukw** [comment]
Dealt with merge conflict.

**Pull request merged**